hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncTool.java

```diff
@@ -190,6 +190,8 @@ protected void doSync() {
             syncHoodieTable(roTableName.get(), false, true);
             // sync a RT table for MOR
             syncHoodieTable(snapshotTableName, true, false);
+            // sync origin table for MOR
+            syncHoodieTable(tableName, true, false);
```

Comment:
Also, can OLAP engine like MySQL query a MOR table with log files?

Reply:
> Also, can OLAP engine like MySQL query a MOR table with log files?

no，only olap engine like starrocks、presto can query mor table with log files @yihua

Reply:
> This can collide with the RO table if the `_ro` suffix is skipped for Hive sync. Could you add a feature flag around the new sync logic?
> 
> ```
> this.roTableName = config.getBoolean(HIVE_SKIP_RO_SUFFIX_FOR_READ_OPTIMIZED_TABLE)
>               ? Option.of(tableName)
>               : Option.of(tableName + SUFFIX_READ_OPTIMIZED_TABLE);
> ```

ok， add a feature for sync origin table @yihua